lib/logstash/filters/grok/timeout_enforcer.rb

```diff
@@ -72,31 +68,16 @@ def start_thread_groking(thread)
     @threads_to_start_time.put(thread, java.lang.System.nanoTime)
```

Comment:
one thing confusing here and in the `grok_till_timeout` method is that the current thread is passed as the `thread` parameter but the current thread is also referenced implicitly using the `java.lang.Thread.xxx`.
AFAIU both should be the same thread so I wonder for clarity sake if we should not use one thread reference notation?
```
java.lang.Thread.interrupted
@threads_to_start_time.put(java.lang.Thread.currentThread(), java.lang.System.nanoTime)
```
or
```
thread.interrupted
@threads_to_start_time.put(thread, java.lang.System.nanoTime)
```
?

Reply:
+1 on standardizing on the instance

Reply:
OK, this is now done